control-plane/roles/logging/README.md

```diff
@@ -0,0 +1,32 @@
+# logging
+
+This role is designed to set up logging using Ansible.
+The role includes tasks to install and configure the following logging tools:
+- Loki-stack
+- Logging ingress for Loki
+
+
+## Variables
+
+This role uses variables from [control-plane-defaults](/control-plane). So, make sure you define them adequately as well.
+
+You can look up all the default values of this role [here](defaults/main/main.yaml).
+The following variables can be set to configure the role:
+
+### General
+
+| Name                                           | Mandatory | Description                                                 |
+|------------------------------------------------|-----------|-------------------------------------------------------------|
```

Comment:
There is a VSCode Plugin which prettyfies markdown tables:

https://marketplace.visualstudio.com/items?itemName=darkriszty.markdown-table-prettify